snippet.js

```diff
@@ -0,0 +1,16 @@
+//function call for update_color(), starting with arbitrary hex color #52beb5
+update_color("#52beb5", 10000, 10000);
```

Comment:
is there a way to code so that a person reading the code knows that this is 10000 MS? Also, I tried changing the value to 1000 so that it can follow the spec; however, it dosnt seem to work.  I believe the product requirement is to change the background color every second for 10 seconds.